### ISSUE TITLE
[logout] logout button is missing. 

### ISSUE THREAD

**Issue by dodikk** [smart-mobile-software/gitstack#48]
I have logged in to the web interface and created some non-admin users.
Now I'm unable to logout and login as that user without having to flush all my cookies in the browser.
I just can't find a "logout" button.

In the other browser I was unable to login as a just created user. I'm sure I've entered a valid password (the one I used to create a user).

**Comment by poiuytrez** [comment]
Thank you. I have scheduled the logout button for the 1.4 milestone.

**Comment by dodikk** [comment]
When are you planning to release 1.4 ?
How about the other problem ( unable to login with a recently created user account )?
Should I create a separate ticket for it?

**Comment by poiuytrez** [comment]
Only the administrator can login to the GitStack administration panel. There is no release date for the next version. It will be released when it will be ready. Everybody knows that software release dates are always delayed :).

**Comment by dodikk** [comment]
Ok. How should the administrator manage ssh keys for created users?
I have created a database and added a user. I've attached the user to that repo as shown on your site http://gitstack.com/wp-content/uploads/2012/02/add_user_repo_full.png

How do I push to that repo?  This is what I get now : 

> > fatal: http://localhost/<repo>.git/info/refs not found : did you run update-server-info on the server? 
> > Same problem for git clone

The site is up and running but I'm unable to access the repo.

Note : "git config user.name" matches with the name of the gitstack user I've created.
